config.py

```diff
@@ -25,21 +25,6 @@
     "verified",
 ]
 
-MEETING_COLUMNS = [
```

Comment:
In the new approach we are pulling all the columns so these are no longer needed.